roles/openshift_master_facts/tasks/main.yml

```diff
@@ -6,7 +6,8 @@
     openshift_master_default_subdomain: "{{ osm_default_subdomain | default(None) }}"
   when: openshift_master_default_subdomain is not defined
 
-- fail:
+- name: Verify required variables are set
+  fail:
```

Comment:
Yes, I'll work on getting that separated out today. This was mainly to unblock further testing.